db/migrate/20170508150135_add_foreign_keys.rb

```diff
@@ -0,0 +1,49 @@
+class AddForeignKeys < ActiveRecord::Migration
+  def change
+    add_index :notifications, [:subject_id, :subject_type]
+    add_index :order_details, :project_id
+    add_index :order_statuses, :parent_id
+    add_index :orders, :merge_with_order_id
+    add_index :price_group_members, :account_id
+    add_index :products, :initial_order_status_id
+
+    # The dartmouth database violates this one
+    add_foreign_key :account_users, :users
```

Comment:
I fixed up Dartmouth's production here. Anything with user_id `114,115,337,168` you can safely delete. That'll probably be quicker than me giving you a fresh snapshot.